tests/runner/Cargo.toml

```diff
@@ -9,10 +9,16 @@ edition = "2021"
 [features]
 default = ["bios", "uefi"]
 bios = ["bootloader/bios"]
-uefi = ["bootloader/uefi", "dep:ovmf-prebuilt"]
+uefi = ["bootloader/uefi"]
 
 [dependencies]
 bootloader = { path = "../..", default-features = false }
 strip-ansi-escapes = "0.1.1"
-ovmf-prebuilt = { version = "0.1.0-alpha.1", optional = true }
 wait-timeout = "0.2.0"
+anyhow = "1.0.89"
+ureq = "2.10.1"
+fs-err = "2.11.0"
+sha2 = "0.10.8"
+tar = "0.4.41"
+lzma-rs = "0.3.0"
+tempfile = "3.12.0"
```

Comment:
Maybe this downloading logic should be added to the ovmf-prebuilt repo instead. There is already a stub crate for this in that repo anyway.

Reply:
Related: https://github.com/rust-osdev/ovmf-prebuilt/issues/38, https://github.com/nicholasbishop/ovmf-prebuilt/commit/73d349224b2f6ab092eb4c736db2aa0e9bed46f3.

Reply:
I agree. Maybe that should be part of a separate PR though.

Reply:
FYI this is available now: https://crates.io/crates/ovmf-prebuilt/0.2.1